react/react1/week2/Todolist/src/components/todoTimer/todoTimer.jsx

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'react';
+
+
+class TodoTimer extends Component {
+    state = {
+        timer: 0
+    }
+
+
+    timerNumber = () => {
+        
+        setInterval(() => {
+            this.setState({
+                timer: this.state.timer++
+            })
+         
+        }, 1000)
+        
+    }
+   
+    componentDidMount() {
```

Comment:
We can use useEffect and the useEffect return statement to get a similar result in a function component. Try rewriting the component